lib/private/converter.rb

```
@@ -84,12 +81,12 @@ def _convert_simple(value, type)
       end
       SafeType::coerce(value, safe_type_rule)
     rescue SafeType::EmptyValueError, SafeType::CoercionError
```

We probably shouldn't rescue error. Let it error out.